tests/test-cases/Group1-Docker-Commands/1-10-Docker-PS.robot

```diff
@@ -187,6 +187,15 @@ Docker ps ports output
     Should Not Contain  ${output}  8002->80/tcp
 
 
+Docker create output with specified static IP
```

Comment:
We need the positive tests as well - confirming that we can set the staticIP as expected when gateway is specified. I do not recall if it’s possible to set a gateway without also specifying an ip range.

Reply:
No sure what’s causing the hang in CI.
Try logging into vic-executor1 from the drone master and you can look at the output.xml as it’s being generated (assuming you cannot recreate locally)

Reply:
I think I get the reason why it get hang. You are missing -d when starting nginx using container-network

Reply:
Done!